internal/node_launcher.sh

```diff
@@ -92,4 +92,8 @@ done
 # Note: for debugging it is useful to see what files are actually present, eg:
 # find . -name thingImLookingFor
 
-exec "${RUNFILES}/TEMPLATED_node" "${NODE_OPTIONS[@]}" "${RUNFILES}/TEMPLATED_script_path" "${ARGS[@]}"
+# Lookup the real paths from the runfiles manifest.
+NODE_PATH=$(grep TEMPLATED_node ${RUNFILES} | awk '{print $2}')
```

Comment:
this doesn't seem very robust & optimized, is there an alternative?

Reply:
It certainly isn't very portable.

/cc @mprobst for ideas

Reply:
Updated to use only bash, and test for equality of the templated bit, so it should not need escaping